c7n/filters/offhours.py

```diff
@@ -376,6 +382,14 @@ def process_resource_schedule(self, i, value, time_type):
             return False
         now = datetime.datetime.now(tz).replace(
             minute=0, second=0, microsecond=0)
+        now_str = now.date().strftime("%Y-%m-%d")
```

Comment:
the date() here is redundant, can just strftime on the datetime

Reply:
Yep, I just used date to make it clear I was only getting date, but I will switch in my next commit.